cli/src/infragpt/shell.py

```
@@ -0,0 +1,270 @@
+                'stdin': slave_fd,
+                'stdout': slave_fd,
+                'stderr': slave_fd,
+                'env': self.env
+            }
+            
+            # Only use preexec_fn on Unix-like systems (not Windows)
+            if hasattr(os, 'setsid'):
+                try:
+                    popen_args['preexec_fn'] = os.setsid
+                except:
+                    pass  # Skip if not supported
```

Ruff (0.12.2)

Use `contextlib.suppress(BaseException)` instead of `try`-`except`-`pass`

(SIM105)